docs/tutorials/_tabs/setting-up-ethernet-over-bluetooth-on-ubuntu-14.04.md

```diff
@@ -30,3 +29,16 @@ If you are using an older version, please upgrade.
     depend on the Bluetooth adapter on your host computer.
 
     {% include screenshot.html source="/images/brickman/bluetooth-confirm-passkey.png" %}
+
+---
+
+These instructions use blueman. Install it with sudo apt-get install blueman. Blueman should work parallel to Network Manager, too. I assume you have already paired your PC with your EV3 brick using the discreption above.
```

Comment:
need backticks ``` ` ``` around `sudo apt-get install blueman` so that it shows up with code formatting.